### PR TITLE
[REM] Accounting: Scrub Taxcloud

### PR DESCRIPTION
**Pull request by tiku-odoo**
Remove TaxCloud from master (16.5) and put admonition warning block in forwarding to Avatax: 

Task: https://www.odoo.com/web#id=3529514&cids=3&model=project.task&view_type=form

### CODE REVIEW

**Review by dade-odoo — APPROVED** [review]
Hey @tiku-odoo ! Not sure if you were explicitly looking for a review from our side or if we just got pinged automatically because it affects accounting docs. Either way, it looks good to me.

File: content/applications/finance/accounting/taxes/taxcloud.rst [hunk 1]
```diff
@@ -2,150 +2,10 @@
...
-the tax computation is correct. Indeed, a 3.5% tax on the full price is the equivalent of a 7% tax
-on half the price, but this might be unexpected from a user point of view.
+   The Odoo TaxCloud integration has begun its decommissioning, starting in Odoo 17. New
+   installations are prohibited in Odoo 17, and in Odoo 18, the TaxCloud module(s) won't exist at
+   all. Odoo recommends the use of the Avatax platform.
 
-.. seealso::
-   :doc:`fiscal_positions`
+   .. seealso::
+      :doc:`avatax`
```
Comment (StraubCreative) [inline content/applications/finance/accounting/taxes/taxcloud.rst:11]: Would separate the **seealso** from the **warning** so it's not embedded visually.
```suggestion
.. seealso::
   :doc:`avatax`
```

**Review by StraubCreative — APPROVED** [review]
This looks good to me, Tim. Thanks!
I had one small suggestion for you below to help improve the warning, visually.

@samueljlieber can you do a technical review here please?
Specifically, since we're removing content I want to be sure there aren't any loose internal links or missing 301s. TIA! 🙏 

cc: @fhe-odoo 

@robodoo delegate=samueljlieber

**Comment by tiku-odoo** [comment]
@samueljlieber This doc is ready for your review. Thanks. 

Tim

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @tiku-odoo nice job! Your changes look good to me. I did not find any other `:doc:` or `:ref:` references to the `taxcloud.rst` doc. I did notice the image folder for `taxcloud.rst` still has images in it– these need to be removed since they are no longer needed in the doc 🙂

I will push up a commit removing the images after my review. Thanks @tiku-odoo!

**Comment by samueljlieber** [comment]
Removed images from `taxcloud/` from [my review](https://github.com/odoo/documentation/pull/5976#pullrequestreview-1657615101) in https://github.com/odoo/documentation/commit/486b6274ced1c42080df5d8753c83e0bb78732a7. All set 🙂

**Review by samueljlieber — APPROVED** [review]

**Comment by samueljlieber** [comment]
@robodoo r+

**Comment by StraubCreative** [comment]
@robodoo r+

**Pull request closed without merging**